src/vaadin-radio-button.html

```diff
@@ -152,6 +154,9 @@
         }
 
         /**
+         * @param {string} name
+         * @param {?string} oldValue
+         * @param {?string} newValue
          * @protected
          */
         attributeChangedCallback(prop, oldVal, newVal) {
```

Comment:
Parameter names are wrong in the `@param` annotations now (copy-paste mistake).
```suggestion
         * @param {string} prop
         * @param {?string} oldVal
         * @param {?string} newVal
         * @protected
         */
        attributeChangedCallback(prop, oldVal, newVal) {
```